kinder/ci/workflows/skew-kubelet-1.17-on-1.18.yaml

```
@@ -0,0 +1,14 @@
+version: 1
+summary: |
+  This workflow tests the proper functioning of kubeadm and control plane from 1.18 against 1.17 kubelet
+  test grid > https://testgrid.k8s.io/sig-cluster-lifecycle-kubeadm#kubeadm-kinder-kubelet-1.17-on-1.18
+  config    > https://git.k8s.io/test-infra/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-kubelet-X-on-Y.yaml
+  config    > https://git.k8s.io/test-infra/testgrid/config.yaml
+vars:
+  kubeadmVersion: "{{ resolve `ci/latest-1.18` }}"
+  kubeletVersion: "{{ resolve `ci/latest-1.17` }}"
+  kubernetesVersion: "{{ resolve `ci/latest-1.18` }}"
+  ignorePreflightErrors: ",KubeletVersion"
```

```suggestion
  ignorePreflightErrors: "KubeletVersion"
```
Let's move the responsibility of adding the initial comma to the template in skew-x-on-y-tasks.yaml

---

i need to check if this is a problem for the trailing `,` for jobs that don't pass the extra `ignorePreflightErrors`.
this may be solvable with go template `if / else` but i don't like it because it's a bit messy.

---

tested with the kubeadm CLI and a trailing `,` seems to be ignored.